client/static/js/board.js

```diff
@@ -0,0 +1,26 @@
+
+var Board = {
+    edges: null,
+    maxRows: 1000,
+    maxCols: 1000,
+
+    init: function(maxRows, maxCols) {
```

Comment:
I think I'm starting with a 10 x 10 board in the backend.  Just something we'll have to keep consistent when we actually instantiate and draw this model